docker-compose.yaml

```
@@ -0,0 +1,26 @@
+version: "3.8"
+
+services:
+  todo_app:
+    container_name: todo_app
+#    build:
+#      context: .
+#      dockerfile: Dockerfile
+    image: todo-app:dev
```

Why not use the dockerfile directly here? It will make the setup instructions much simpler (i.e. just running `docker compose up` without a build instruction):
```suggestion
    build:
      context: .
      target: development
```